src/Microsoft.Graph.Core/Requests/Middleware/RetryHandler.cs

```diff
@@ -200,13 +150,12 @@ private void AddOrUpdateRetryAttempt(HttpRequestMessage request, int retry_count
         /// <param name="retry_count">The retry counts</param>
         /// <param name="delay">Delay value in seconds.</param>
         /// <param name="delayInSeconds"></param>
-        /// <param name="cumulativeDelay">The cumulative retry time </param>
         /// <param name="cancellationToken">The cancellationToken for the Http request</param>
         /// <returns>The <see cref="Task"/> for delay operation.</returns>
-        public Task Delay(HttpResponseMessage response, int retry_count, int delay, out double delayInSeconds, ref TimeSpan cumulativeDelay, CancellationToken cancellationToken)
+        public Task Delay(HttpResponseMessage response, int retry_count, int delay, out double delayInSeconds, CancellationToken cancellationToken)
```

Comment:
This shouldn't be `public`. This should be at least `internal`, ideally `private`.

Reply:
Using `internal` as this is accessed from one of the test class methods.
https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/2c23a8bdeeaa608e7e5e36494b443e5982fbef6b/tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/Middleware/RetryHandlerTests.cs#L276-L284